larray/viewer/model.py

```diff
@@ -227,14 +223,27 @@ def _set_data(self, data, xlabels, ylabels, changes=None, bg_gradient=None, bg_v
 
         assert isinstance(changes, dict)
         self.changes = changes
-        self._data = data
-        if xlabels is None:
-            xlabels = [[], []]
-        self.xlabels = xlabels
-        if ylabels is None:
-            ylabels = [[]]
+
+        # convert input data to LArray if not
+        # add extra dimension if 1D
+        la_data = la.aslarray(data)
+        if la_data.ndim == 1:
+            la_data = la_data.reshape([la.Axis(1), la_data.axes[0]])
```

Comment:
I am a bit nervous about this. I would prefer to keep the larray as it was given and reshape the underlying numpy data (in the "Convert LArray data to a 2D Numpy array" step below) so that we can export it directly without it having an extra dimension